src/bic/collection/form-records.js

```diff
@@ -33,14 +33,14 @@ define(function (require) {
 
             collection.reset();
 
-            nodes = data.evaluate('//' + formName, data);
+            nodes = data.evaluate('//' + formName, data, null, XPathResult.ANY_TYPE, null);
             node = nodes.iterateNext();
 
             parseNodes = function (key) {
               if (key.nodeName === 'id') {
-                parsed.id = key.innerHTML;
+                parsed.id = key.textContent;
```

Comment:
It's probably worth have a unit test that uses a human-friendly XML file with new-lines and other white-space.